README.md

```diff
@@ -375,6 +376,19 @@ Give your images a shadow and rounded corners.
 
 ---
 
+## [Media Grid](https://github.com/zremboldt/obsidian-media-grid)
+
+by [@zremboldt](https://github.com/zremboldt)
+
+Add more meaning and visual interest to your documents by easily grouping sets of images, videos, or even audio tracks into grid layouts.  
+I've linked to my repo for a more thorough explanation of custom settings, requirements, how it works, etc.
+
+![](media/css-snippets/media-grid.png)
+
+[Obsidian Media Grid repo](https://github.com/zremboldt/obsidian-media-grid)
```

Comment:
Follow the conventions of `[📁 filename.css](<link to snippet>)` and add [this file](https://github.com/zremboldt/obsidian-media-grid/blob/main/media-grid-snippet.css) inside `code/css-snippets`.